### ISSUE TITLE
Mod loading error has occurred

### ISSUE THREAD

**Issue by QianFuv** [AceTheEldritchKing/Secrets-Of-Forging-Revelations#4]
https://mclo.gs/R1F4Qur(A few months old crash report)
https://mclo.gs/gRBGyoO(Latest crash report)

**Comment by AceTheEldritchKing** [comment]
Hello,
Does this happen when you start the server? Does it happen on singleplayer as well?

**Comment by QianFuv** [comment]
> Hello,
> Does this happen when you start the server? Does it happen on singleplayer as well?

no just happen when I launch the game

**Comment by AceTheEldritchKing** [comment]
That's odd, I've had a similar issue that was fixed by just restarting the game. Has that worked or not? Do you have Art of Forging installed, and if you do, have you gotten the same issue with it? The way effect bars also almost set up the same way in both mods.

**Comment by QianFuv** [comment]
Yes, restarting can solve it, but this is indeed an error, isn't it? I have installed all the subsidiary mods of Tetra on 1.19.2 Forge

**Comment by AceTheEldritchKing** [comment]
It is an error, however, at a deeper level I am unsure of a fix as restarting the client resolves the issue. The issue is on client side regarding the display of the effect bars in the workbench and holosphere.

**Comment by QianFuv** [comment]
I hope it can be fixed, because this unstable collapse is the most tormenting

**Comment by AceTheEldritchKing** [comment]
Due to Tetra being closed sourced and my departure from creating addons for it, I don't believe a permanent solution will come soon, although I will do my best to deliver when I can.
